dashboard/app/controllers/api/v1/pd/workshops_controller.rb

```diff
@@ -11,7 +11,7 @@ def index
   end
 
   def k5_public_map_index
-    @workshops = @workshops.where(
+    @workshops = Pd::Workshop.where(
```

Comment:
It was meant to be public all along, but yeah that's a fair point. I'll prepare another PR with a limited serializer.